src/parse/swissDrivingLicenseFields.ts

```diff
@@ -17,72 +17,83 @@ import {
   firstNameTemplate,
 } from './fieldTemplates';
 
-export default [
-  {
-    ...documentNumberTemplate,
-    line: 0,
-    start: 0,
-    end: 9,
-    parser: parseDocumentNumber,
-  },
-  {
-    label: 'Language code',
-    field: 'languageCode',
-    line: 0,
-    start: 6,
-    end: 7,
-    parser: parseLanguageCode,
-    type: fieldTypes.ALPHABETIC,
-  },
-  {
-    ...documentCodeTemplate,
-    line: 1,
-    start: 0,
-    end: 2,
-    parser: parseDocumentCode,
-  },
-  {
-    ...issuingStateTemplate,
-    line: 1,
-    start: 2,
-    end: 5,
-    parser: parseIssuingState,
-  },
-  {
-    label: 'PIN code',
-    field: 'pinCode',
-    line: 1,
-    start: 5,
-    end: 14,
-    parser: parseNumber,
-    type: fieldTypes.NUMERIC,
-  },
-  {
-    label: 'Version number',
-    field: 'versionNumber',
-    line: 1,
-    start: 14,
-    end: 17,
-    parser: parseNumber,
-    type: fieldTypes.NUMERIC,
-  },
-  {
-    label: 'Separator 1',
-    field: null,
-    line: 1,
-    start: 17,
-    end: 19,
-    parser: checkSeparator,
-  },
-  { ...birthDateTemplate, line: 1, start: 19, end: 25 },
-  {
-    label: 'Separator 2',
-    field: null,
-    line: 1,
-    start: 25,
-    end: 30,
-    parser: checkSeparator,
-  },
-  { ...lastNameTemplate, line: 2, start: 0, end: 30 },
-  { ...firstNameTemplate, line: 2, start: 0, end: 30 },
-].map(createFieldParser);
+export default function swissDrivingLicenseFields(lines: string[]) {
+  const names = lines[2].replace(/^<+|<+$/g, '').split('<<');
```

Comment:
This is very risky as it only works if the line is formatted correctly

Reply:
@targos It is acceptable to use "<<" as the delimiter between the first and last name?

Reply:
Yes, but the parsing should be resilient to lines that don't follow the standard.
- What about lines without any `<<`?
- What about lines with multiple names separated by two `<<`?

I would prefer not to fundamentally change the parser to accomodate an edge case related to autocorrect only.

Also, this pattern isn't specific to the Swiss driving license. Don't we need similar code changes for other MRZ formats?